swift/example_code/s3/DeleteObjects/Tests/DeleteObjectsTests/ServiceHandler_Ext.swift

```diff
@@ -0,0 +1,103 @@
+/*
+   Extensions to the `ServiceHandler` class to handle tasks we need
+   for testing that aren't the purpose of this example.
+
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+import Foundation
+import AWSS3
+import AWSClientRuntime
+import ClientRuntime
+import SwiftUtilities
+@testable import ServiceHandler
+
+public extension ServiceHandler {
+    /// Create a new user given the specified name.
+    ///
+    /// - Parameters:
+    ///   - name: Name of the bucket to create.
+    /// Throws an exception if an error occurs.
+    func createBucket(name: String) async throws {
+        let config = S3ClientTypes.CreateBucketConfiguration(
+            locationConstraint: self.region
+        )
+        let input = CreateBucketInput(
+            bucket: name,
+            createBucketConfiguration: config
+        )
+        _ = try await client.createBucket(input: input)
+    }
+
+    /// Delete a bucket.
+    /// - Parameter name: Name of the bucket to delete.
+    func deleteBucket(name: String) async throws {
+        let input = DeleteBucketInput(
+            bucket: name
+        )
+        _ = try await client.deleteBucket(input: input)
+    }
+
+    /// Create a file in the specified bucket with the given name. The new
+    /// file's contents are uploaded from a `Data` object.
+    ///
+    /// - Parameters:
+    ///   - bucket: Name of the bucket to create a file in.
+    ///   - key: Name of the file to create.
+    ///   - data: A `Data` object to write into the new file.
+    func createFile(bucket: String, key: String, withData data: Data) async throws {
+        let dataStream = ByteStream.from(data: data)
+
+        let input = PutObjectInput(
+            body: dataStream,
+            bucket: bucket,
+            key: key
+        )
+        _ = try await client.putObject(input: input)
+    }
+
+    /// Deletes the specified file from Amazon S3.
```

Comment:
Deletes --> Delete

Reply:
I disagree on this one, since it's the documentation comment for the entire function; for a style match with the rest of the content and usage in other AWS documentation, it should be "Deletes".